bear-requirements.txt

```diff
@@ -29,3 +29,4 @@ vim-vint~=0.3.10
 vulture~=0.10.0
 yamllint~=1.5
 yapf~=0.14.0
+dependency_management~=0.2.0
```

Comment:
Please don't add this, it can cause all kinds of dependency problems as we've already inherited it from coala. I see you have a pull there, Please remove this when it gets accepted.